api/candlepin-api-spec.yaml

```diff
@@ -7166,7 +7166,9 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/AsyncJobStatusDTO'
+                type: array
```

Comment:
I was getting an issue with the generated API class failing to deserialize because Candlepin is returning a list and the generated API is expecting a single item json.